omero/sysadmins/server-advanced-configuration.rst

```diff
@@ -7,9 +7,7 @@ In some cases, the configuration properties will not suffice to fully
 configure your server. In that case, it may be necessary to make use of
 IceGrid's XML configuration files. Like the :file:`config.xml` file mentioned
 above, these are stored under :file:`etc/grid`:
-:file:`etc/grid/default.xml` is used on Unix systems,
-:file:`etc/grid/windefault.xml` is used on
```

Comment:
@joshmoore: Should we get rid of this file?

Reply:
I'd think so.

Reply:
Created https://trello.com/c/Zj6iJqHc/780-remove-traces-of-windows-server - I might have missed something related.